src/apps/collectionsUi/viewModules/docs/__markdown__/overview/general.en.md

```diff
@@ -0,0 +1,4 @@
+# THIS IS AN OVERIVEW IN MARKDOWN
```

Comment:
Fix OVERIVEW => OVERVIEW, as this might be the first thing people see

Reply:
sure